docs/Config.md

```diff
@@ -305,30 +456,32 @@ os:
 
 ```yaml
 os:
-  open: 'xdg-open {{filename}} >/dev/null'
+  open: "xdg-open {{filename}} >/dev/null"
```

Comment:
It looks like you pressed Command-S in VS Code to get these reformatted. Happens to me all the time...

I think you should either drop these changes (all the ones from here on down), or make a separate commit (before this one) that reformats the file. I could go either way, but I'm not sure if @jesseduffield has a strong preference of using single quotes over double quotes, so I'll let him decide what to do.

Reply:
Woops :laughing: I dropped it out of the commit.